spoon/tests/SpoonTest.php

```diff
@@ -16,11 +16,9 @@ public function testGet()
 		$this->assertEquals(Spoon::get('stored_value'), $value);
 	}
 
-	/**
-	 * @expectedException SpoonException
-	 */
 	public function testGetFailure()
 	{
+	    $this->expectException(SpoonException::class);
```

Comment:
Klopt de identing?

Reply:
fixed in 3b4f30a2668a44a72aee04d685cf0f289aef1ae0